package/MDAnalysis/analysis/encore/covariance.py

```diff
@@ -215,7 +215,7 @@ def covariance_matrix(ensemble,
         # Select the same atoms in reference structure
         reference_atom_selection = reference.select_atoms(
             ensemble.get_atom_selection_string())
-        reference_coordinates = reference_atom_selection.atoms.coordinates()
+        reference_coordinates = reference_atom_selection.atoms.positions
```

Comment:
Could you please add a test that check this fix works?

Reply:
can you tell me please, how to add test?

Reply:
@manishsaini6421 please write a function with code that uses this pathway. It should fail with the original `.coordinates()` and succeed with the new `.positions`. Have a look at the tests in https://github.com/MDAnalysis/mdanalysis/blob/develop/testsuite/MDAnalysisTests/analysis/test_encore.py and more generally throughout https://github.com/MDAnalysis/mdanalysis/tree/develop/testsuite/MDAnalysisTests for ideas :)